examples/sql_bank/main.go

```diff
@@ -49,15 +49,17 @@ func moveMoney(db *sql.DB) {
 		if err != nil {
 			log.Fatal(err)
 		}
-		if _, err = tx.Exec("SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
+		if _, err := tx.Exec("SET TRANSACTION ISOLATION LEVEL SERIALIZABLE"); err != nil {
 			log.Fatal(err)
 		}
-		rows, err := tx.Query("SELECT id, balance FROM accounts WHERE id IN ($1, $2)", from, to)
-		if err != nil {
+		var rows *sql.Rows
+		if r, err := tx.Query("SELECT id, balance FROM accounts WHERE id IN ($1, $2)", from, to); err == nil {
```

Comment:
while this is one of the simpler modifications, it's a great example of how readability is reduced.
If we decide to ban err shadowing from here on then ok, otherwise this is really not an improvement.